csm/core/agent/api.py

```diff
@@ -560,7 +529,9 @@ async def process_websocket(request):
     async def _on_startup(app):
         Log.debug('REST API startup')
         CsmRestApi._bgtasks.append(app.loop.create_task(CsmRestApi._websock_bg()))
-        CsmRestApi._bgtasks.append(app.loop.create_task(CsmRestApi._ssl_cert_check_bg()))
+
+        # Commented as securityservice and provisioner plugin is archieved.
+        # CsmRestApi._bgtasks.append(app.loop.create_task(CsmRestApi._ssl_cert_check_bg()))
```

Comment:
Minor: "~~is archieved~~ are archived". why is this comment needed? maybe, just remove these 2 lines?

Reply:
updated the comment.